frontend/src/container/TopNav/DateTimeSelection/index.tsx

```diff
@@ -169,6 +173,11 @@ function DateTimeSelection({
 		return `Last refresh - ${secondsDiff} sec ago`;
 	}, [maxTime, minTime, selectedTime]);
 
+	const isLogsExplorerPage = useMemo(
```

Comment:
marking an explicit approval but need to fix the root cause

Reply:
https://github.com/SigNoz/signoz/issues/4233 Will investigate here

Reply:
Resolving the comment